src/flags/flagsFactory.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-prototype-builtins */
+
+function flagValue(bridgeState, flagHeights, flag) {
+  if (!flagHeights.hasOwnProperty(flag)) {
+    return true;
+  }
+
+  const targetHeight = flagHeights[flag];
+  return bridgeState.blockHeight >= targetHeight;
```

Comment:
`bridgeState.blockHeight` is updated only every 32 blocks

https://github.com/leapdao/leap-node/blob/d7280b3ce39fc9cdf9ebee47247442d236743ae4/src/block/index.js#L44

Reply:
if a single 32-blocks chunk has both types of txs, how to make a switch?

Reply:
In `saveState` i see only `bridgeState.currentState.blockHeight` update. For flags i use `bridgeState.blockHeight` which updates on each block.

Reply:
Am i miss something?

Reply:
no, all is great 👍

Confused with `this.currentState.blockHeight`